src/frida/tracer.py

```diff
@@ -817,6 +846,8 @@ def process_builder_arg(option, opt_str, value, parser, method, **kwargs):
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.include_imports,))
             parser.add_option("-m", "--include-objc-method", help="include OBJC_METHOD", metavar="OBJC_METHOD",
                     type='string', action='callback', callback=process_builder_arg, callback_args=(pb.include_objc_method,))
+            parser.add_option("-s", "--include-symbol", help="include DEBUG_SYMBOL", metavar="DEBUG_SYMBOL",
+                    type='string', action='callback', callback=process_builder_arg, callback_args=(pb.include_debug_symbol,))
```

Comment:
We should either drop the `debug_`/`DEBUG_` qualifiers here or always use them. On a second thought perhaps `--include-debug-symbol` is clearer and we should just go for that. (Like you initially suggested.)